src/Components/Patient/PatientHome.tsx

```diff
@@ -1511,33 +1440,35 @@ export const PatientHome = (props: any) => {
           Consultation History
         </h2>
         {consultationList}
-        {!isConsultationLoading && totalConsultationCount > limit && (
-          <div className="mt-4 flex w-full justify-center">
-            <Pagination
-              cPage={currentConsultationPage}
-              defaultPerPage={limit}
-              data={{ totalCount: totalConsultationCount }}
-              onChange={handleConsultationPagination}
-            />
-          </div>
-        )}
+        {(!isConsultationLoading && consultationListData?.count) ||
+          (0 > limit && (
+            <div className="mt-4 flex w-full justify-center">
+              <Pagination
+                cPage={currentConsultationPage}
+                defaultPerPage={limit}
+                data={{ totalCount: consultationListData?.count || 0 }}
+                onChange={handleConsultationPagination}
+              />
+            </div>
+          ))}
       </div>
 
       <div>
         <h2 className="my-4 ml-0 text-2xl font-semibold leading-tight">
           Sample Test History
         </h2>
         {sampleList}
-        {!isSampleLoading && totalSampleListCount > limit && (
-          <div className="mt-4 flex w-full justify-center">
-            <Pagination
-              cPage={currentSampleListPage}
-              defaultPerPage={limit}
-              data={{ totalCount: totalSampleListCount }}
-              onChange={handleSampleListPagination}
-            />
-          </div>
-        )}
+        {(!isSampleLoading && sampleListData?.count) ||
+          (0 > limit && (
+            <div className="mt-4 flex w-full justify-center">
+              <Pagination
+                cPage={currentSampleListPage}
+                defaultPerPage={limit}
+                data={{ totalCount: sampleListData?.count || 0 }}
+                onChange={handleSampleListPagination}
+              />
+            </div>
+          ))}
```

Comment:
Why was this limit is positive check added?

Avoid this check since we are hardocding the `limit` above as 5. So we know for sure the limit is always greater than 0.


```suggestion
          (
            <div className="mt-4 flex w-full justify-center">
              <Pagination
                cPage={currentSampleListPage}
                defaultPerPage={limit}
                data={{ totalCount: sampleListData?.count || 0 }}
                onChange={handleSampleListPagination}
              />
            </div>
          )}
```